experimental/dds/tree2/src/shared-tree/sharedTree.ts

```diff
@@ -201,10 +205,21 @@ export class SharedTree
 		config: InitializeAndSchematizeConfiguration<TRoot>,
 	): SharedTreeView {
 		// TODO:
-		// This should work, but schema editing on views doesn't send ops.
-		// this.view.schematize(config);
-		// For now, use this as a workaround:
-		schematizeView(this.view, config, this.storedSchema);
+		// When this becomes a more proper out of schema adapter, editing should be made lazy.
+		// This will improve support for readonly documents, cross version collaboration and attribution.
+
+		// Check for empty.
+		// TODO: Better detection of empty case
```

Comment:
If we plan on tackling this soon, it's probably fine as is. Otherwise, might be worth adding a bit more detail here about why the current check is insufficient, and what the proposed solution is (if we have one).

Reply:
note that these are not new comments, they were simple moved along with the code they are commenting on. This has been somewhat improved since this was writtem, so removing its prabably fine.

```suggestion
```